### PR TITLE
[2.42.2] Integration API GET /console/v1/auth route

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: content/api/integration-api/console-api.md [hunk 1]
```diff
@@ -96,7 +96,37 @@ Option value (string)
...
+
+#### Example
+
+```
+$ curl -k $REPLICATED_INTEGRATIONAPI/console/v1/auth | jq
+{
+  "authType": "ldap",
+  "form": {
+    "password": "",
+    "username": ""
```
Comment (laverya) [inline content/api/integration-api/console-api.md]: this is scary
can we somehow clarify that these will _always_ be the empty string, and it's the keys that matter?

Comment (emosbaugh) [reply content/api/integration-api/console-api.md]: how about this or do you have a suggestion?

```
{
  "authType": "ldap",
  "fields": ["usename", "password"]
}
```

Comment (emosbaugh) [reply content/api/integration-api/console-api.md]: or 

```
{
  "authType": "ldap",
  "form": [
    {"name": "password", "type": "string", "title": "Password"},
    {"name": "username", "type": "string", "title": "Username"}
}

Comment (laverya) [reply content/api/integration-api/console-api.md]: either would be fine with me
I was thinking more of the "add a note to the docs here to clarify things" style

Comment (emosbaugh) [reply content/api/integration-api/console-api.md]: see the next commit. i tried to clarify it in the description of the param

**Pull request merged**